src/guides/v2.3/javascript-dev-guide/widgets/widget_gallery.md

```diff
@@ -580,3 +580,56 @@ api.updateOptions([{
 [Fotorama widget]: http://fotorama.io/
 [lib/web/mage/gallery/gallery.js]: {{ site.mage2bloburl }}/{{ page.guide_version }}/lib/web/mage/gallery/gallery.js
 [lib/web/magnifier/magnify.js]: {{ site.mage2bloburl }}/{{ page.guide_version }}/lib/web/magnifier/magnify.js
+
+## Code sample
+
+This example shows a use case for a gallery widget on any page.
```

Comment:
```suggestion
This example shows a use case for a gallery widget on any page:
```

Reply:
Hi @BarnyShergold 
All your suggestions have been implemented!

Thank you!